### PR TITLE
COP 1977: Resize components (2)

### PR DESCRIPTION
**Pull request by JimmySunDance**
## Description
Resize UI to fit surface Go 2 screen

## To test
- Run on surface Go 2. 
- Elements should display at a reasonable size. 

## Developer Checklist

\* Required

- [ ] Up-to-date with master branch? \*

- [ ] Does it have tests?

- [x] Pull request URL linked to JIRA ticket? \*

- [ ] All reviewer comments resolved/answered? \*

### CODE REVIEW

File: packages/react/src/config/camera.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const livePhotoConfig = {
   video: {
-    width: 926,
+    width: 720,
```
Comment (Ali-Baker) [inline packages/react/src/config/camera.js:3]: 👏 Pefect

Comment (JimmySunDance) [reply packages/react/src/config/camera.js:3]: 😉

File: packages/react/src/Components/Pages/index.jsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ const Index = () => (
           <ImageComparisonsTable />
           <hr className="govuk-section-break govuk-section-break--m" />
           <DataReadTable />
-          <p>
+          <p className="govuk-body font--26">
```
Comment (Ali-Baker) [inline packages/react/src/Components/Pages/index.jsx:28]: What's `govuk-body`?

**Review by Ali-Baker — APPROVED** [review]
Apart from `govuk-body` which may be used correctly. I am with this.

**Pull request merged**